cli/lib/kontena/cli/subcommand_loader.rb

```diff
@@ -54,6 +59,7 @@ def klass
       unless safe_require(path) || safe_require(Kontena.cli_root(path))
         raise ArgumentError, "Can't load #{path} or #{Kontena.cli_root(path)}"
       end
+      warn "[DEPRECATED] Command has been deprecated: #{deprecated}" if deprecated?
```

Comment:
Should this deprecation warning happen as part of `Kontena::Command` instead? Thinking that the subcommand loader is the wrong place to start extending clamp commands in this way, and this will get in the way if we e.g. implement some dynamic command completer stuff in the future?